packages/block-library/src/block/index.php

```diff
@@ -22,6 +24,21 @@ function render_block_core_block( $attributes ) {
 		return '';
 	}
 
+	if ( in_array( $attributes['ref'], $seen_refs, true ) ) {
+		trigger_error(
```

Comment:
Should we also show in in admin? I think the `Warning` is enough there.
Maybe trigger if `! is_admin()`?

Reply:
Good catch! On my end, the warning was hidden by Gutenberg's full screen mode.

Reply:
→ 90eb5866c4